packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -553,6 +553,8 @@ class _MenuAnchorState extends State<MenuAnchor> {
   }
 
   void _closeChildren({bool inDispose = false}) {
+    // todo: remove when interpolated `$this` parsing is fixed (https://github.com/dart-lang/sdk/issues/50263)
```

Comment:
That issue is closed, can the code just be fixed now and the ignore be removed?

Reply:
Side-note: I am surprised that this line didn't trigger the `flutter_style_todos` lint as this `todo` is not following our style: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions. If we need to keep this TODO, could you bring it up to style?

Reply:
The issue is closed but the fix doesn't seem to have made it into an engine roll yet.  (I was holding out for that but things are piling up behind this release.)  Annoying for sure...

I can update the TODO format for sure.

Reply:
OK, I understand. If you could update the TODO format, this is good then.

Reply:
Thanks. Done. 👍